<launch>
    <param name="/gaze_pose/model_path" 
    value="$(find face_processor)/src/face_processor/intel/gaze-estimation-adas-0002/FP32/gaze-estimation-adas-0002"/>
    
    <node pkg="face_processor"
    type="gaze_pose_node.py"
    name="gaze_pose"
    output="screen"/>
    
</launch>